Microsoft.Toolkit.Win32.WpfCore.SampleApp/Microsoft.Toolkit.Win32.WpfCore.SampleApp.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UseWPF>true</UseWPF>
+    <ApplicationIcon>Office.ico</ApplicationIcon>
```

Comment:
Don't pack it, or else it will fail the Pack step since its nuget were not generated, and we don't want to generate a nuget out of this sample anyway hehe
```suggestion
    <ApplicationIcon>Office.ico</ApplicationIcon>
    <IsPackable>false</IsPackable>
```